frontend/src/component/TabBar/TabBar.styl

```
@@ -0,0 +1,46 @@
+@import "~tracim_frontend_lib/src/css/Variable.styl"
+
+tabWidth = 185 //185px
+
+.tabBar
+  display flex
+  background-color offWhite
+  & .pageTitleGeneric
+    width "calc(100% - %s)" % (3*tabWidth)px
```

```suggestion
    width "calc(100% - %s)" % (3*@width)px
```

(so this hopefully does not need to be duplicated in the `@media` block below - I don't know if it actually works)

---

Updated